tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -221,6 +227,28 @@ Verify User Can Launch A Workbench
     Check Launched Workbench Is The Correct One     workbench_title=${WORKBENCH_2_TITLE}
     ...    image=${NB_IMAGE}    namespace=${ns_name}
 
+Verify User Can Add A S3 Data Connection
```

Comment:
I would change the name to explicitly mention the fact that the data connection has to be connected to a Workbench

Reply:
done